policy/lib/openshift.rego

```diff
@@ -38,4 +38,24 @@ is_pod_or_networking {
 
 is_pod_or_networking {
     is_route
+}
+
+is_policy_active(policyId) {
+    not konstraint_core.is_gatekeeper
+}
+
+is_policy_active(policyId) {
+    konstraint_core.is_gatekeeper
+
+    not contains(namespace_disabled_policies_label, policyId)
```

Comment:
Instead of `contains`, I would allow multiple policies to be specified in this label with a separator value. Split on the separator and then do the comparison. This would allow greater flexibility and potentially avoid matching an undesired policy